mappings/net/minecraft/client/gui/components/debugchart/AbstractDebugChart.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_bkzzefxr net/minecraft/client/gui/components/debugchart/AbstractDebugChart
```

Comment:
so --
I haven't looked deep in here, but I assume it's stuff like the f3 pie, which would have me place it in `gui/hud` instead of `/components/debugchart`, which creates two brand new packages